heltour/tournament/views.py

```diff
@@ -1213,6 +1218,51 @@ def _view(league_tag, season_tag, user_data):
         return _view(self.league.tag, self.season.tag, self.user_data)
 
 
+class ActivePlayerTableView(LeagueView):
+    @cached_as(League, Season, Round)
+    def view(self, page: int=1):
+        black_games = self.games_as_black()
+        white_games = self.games_as_white()
+        seasons = Count("seasonplayer", filter=Q(seasonplayer__season__league=self.league), distinct=True)
+        newest_seasons = SeasonPlayer.objects.filter(season__league=self.league, player=OuterRef("pk")).order_by("-season__start_date")
+
+        tablesums = Player.objects.annotate(blackcount=black_games) \
+                .annotate(whitecount=white_games) \
+                .annotate(game_count=F("blackcount")+F("whitecount")) \
+                .annotate(season_count=seasons) \
+                .annotate(latest_season=Subquery(newest_seasons.values("season__tag")[:1])) \
+                .order_by("-game_count", "season_count")
+        paginator = Paginator(tablesums, DEFAULT_PAGE_SIZE)
+
+        page_obj = paginator.get_page(page)
+
+        context = {
+                'page_obj': page_obj,
+        }
+
+        return self.render('tournament/active_players.html', context)
+
+    def games_as_black(self):
+        if self.league.is_team_league():
+            return Count("pairings_as_black",
+                    filter=Q(pairings_as_black__teamplayerpairing__team_pairing__round__season__league=self.league) & ~Q(pairings_as_black__teamplayerpairing__game_link=""),
```

Comment:
The `Count()` and the `distinct=True` should be above, in the view, as it goes with the annotate calls